### ISSUE TITLE
Saving all the Trajectory if not Merging

### ISSUE THREAD

**Issue by ayushGHub** [UZ-SLAMLab/ORB_SLAM3#361]
Is it possible to save all the trajectories generated instead of the only one which is saved automatically after mapping ends?

This image sequence of walking on a straight road so they will never reach any points mapped before so there won't be any merging. So I want to save all the trajectories.

This is how it looks like
![12](https://user-images.githubusercontent.com/29687795/128895160-70388733-52f3-4188-949f-ef14c81dfd08.png)

**Comment by wangtsiao** [comment]
hi  ayushGHub, this question have been mentioned here https://github.com/UZ-SLAMLab/ORB_SLAM3/pull/310.

maybe you can try this fixed repository https://github.com/neo-knight-td/ORB_SLAM3/tree/save_load_branch.

**Comment by Jiaxin8122** [comment]
Could we add wechat (william_hjx)? Just few questions about the save/load function. Thx

**Comment by wangtsiao** [comment]
@Jiaxin8122 hi, jiaxin8122 welcome to post question here.

**Comment by wangtsiao** [comment]
Yeah, don't need to save mapPoints' id. Since mapPoints's points save in a STL vector, we directly save MapPoints though `ar &mvpMapPoints;` in serialize(). As for why the status is lost, I don't know, may be it's  a bug in ORB_SLAM3. In short, ORB_SLAM is a repository that needs to be developed.
